cips/cip-template.md

```diff
@@ -42,6 +42,18 @@ It is recommended to follow RFC 2119 and RFC 8170. Do not remove the key word de
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
 
+## Parameters
+
+The parameters section should summarize any changes to global or module parameters, including any new parameters, introduced by the CIP. All mainnet parameters are tracked in [CIP-13](./cip-13.md). Once a CIP is accepted and deployed to mainnet, CIP-13 MUST be updated with these parameter changes. If there are no parameter changes in the CIP, this section can be omitted. 
+
+**TODO: Remove the previous comments and update the following table before submitting**
+
+| Parameter     | Default | Summary                                                                                                                | Changeable via Governance |
+|---------------|---------|------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| module1.Name1 | Value1  | Description1                                                            | bool                     |
+| module2.Name2 | Value2  | Description2                                                            | bool                     |
+
+
```

Comment:
```suggestion
For changes to existing parameters, use the following table:

| Parameter           | Current value | Proposed value | Description    | Changeable via Governance |
|-----------------|---------------|-----------------|--------------|-----------------------------|
| module1.Name1 | CurrentValue1 | ProposedValue1 | Description1 | bool                                          |
| module2.Name1 | CurrentValue2 | ProposedValue2 | Description2 | bool                                          |

For new parameters the Current value column can be omitted, using the following table:

| Parameter          | Proposed value | Description        | Changeable via Governance |
|-----------------|----------------|-----------------|-----------------------------|
| module1.Name1 | Value1                | Description1      | bool                                         |
| module2.Name2 | Value2              | Description2      | bool                                         |
```

Reply:
@rootulp do you think it makes sense to include both examples?

Reply:
I'm not strongly opinionated. I think this section could just contain the first example table with a line after that says the Current value column can be omitted. E.g.

```suggestion
For changes to existing parameters, use the following table:

| Parameter     | Current value | Proposed value | Description  | Changeable via Governance |
|---------------|---------------|----------------|--------------|---------------------------|
| module1.Name1 | CurrentValue1 | ProposedValue1 | Description1 | bool                      |
| module2.Name1 | CurrentValue2 | ProposedValue2 | Description2 | bool                      |	
	
For new parameters the Current value column can be omitted.
```

Reply:
sounds good to me and good idea